README.md

```diff
@@ -1,31 +1,207 @@
 
-```bash 
-pip install kody-clientsdk-python
+If your coding language is not listed, please let the Kody team know and we will be able to create it for you.
+
+## Authenticate to Payments API
+
+The client library uses a combination of a `Store ID` and an `API key`. These will be shared with you during the technical integration onbiarding or by your Kody contact.
+During develoment you will have access to a test Store and test API key, and when the integration is ready for live access, those production credentials will be shared securely.
+Test and live API calls are always compatible, only changing credentials and the service hostname is required.
+
+### Host names
+
+- Development and test: `https://grpc-staging.kodypay.com`
+- Live: `https://grpc.kodypay.com`
+
+## Payments API Documentation
+
+Kody supports the following channels to accept payments via API.
+
+1. Terminal - In-person payments
+2. Online - E-commerce payments
+
+Each of these channels have their own collection of services that are available as method calls on the client library:
+- `KodyPayTerminalService`
+- `KodyEcomPaymentsService`
+
+### Terminal
+
+The Kody Payments API Terminal service has the following methods:
+
+- `KodyPayTerminalService.Terminals` - returns all the terminals of the store and their online status
+- `KodyPayTerminalService.Pay` - initiate a terminal payment
+- `KodyPayTerminalService.Cancel` - cancel an active terminal payment
+- `KodyPayTerminalService.PaymentDetails` - get the payment details
+
+### Online
+
+The Kody Payments API Online service has the following methods:
+
+- `KodyPayTerminalService.Terminals` - returns all the terminals of the store and their online status
+- `KodyPayTerminalService.Pay` - initiate a terminal payment
+- `KodyPayTerminalService.Cancel` - cancel an active terminal payment
+- `KodyPayTerminalService.PaymentDetails` - get the payment details
+
+
+## Demo code
+
+### **Get list of Terminals**
+
+This simple call is a fast, read only method, that returns a list of all terminals assigned to the store, and their online status.
+
+````python
+ with grpc.secure_channel(target=config.address,
+                             credentials=grpc.ssl_channel_credentials()) as channel:
+        stub = pay_grpc_client.KodyPayTerminalServiceStub(channel)
+        response = stub.Terminals(
+            pay_model.TerminalsRequest(store_id=config.store_id),
+            metadata=[("x-api-key", config.api_key)]
+        )
+````
+- TerminalResponse : Terminal Response
+
+```python
+message TerminalsResponse {
+  repeated Terminal terminals = 1;
+}
+
+message Terminal {
+  string terminal_id = 1; // terminal serial number
+  bool online = 2;
+}
 ```
 
+### **Create terminal payment**
+
+Send to a terminal a payment initiation request. This request can make the terminal to immediately display a card acquiring screen, or display a tip screen to the user which after will go to the card scanning screen.
```

Comment:
```suggestion
Send a payment initiation request to a terminal. This request will either make the terminal immediately display the card acquiring screen, or display a tip screen to the user after which it will go to the card acquiring screen.
```